simpleclient_caffeine/src/test/java/io/prometheus/client/cache/caffeine/CacheMetricsCollectorTest.java

```diff
@@ -70,6 +70,7 @@ public void loadingCacheExposesMetricsForLoadsAndExceptions() throws Exception {
         }
         cache.get("user3");
 
+
```

Comment:
nit: Probably could be removed this line change?

Reply:
Done